### PR TITLE
Update in-app links to new organization name

### LINKED ISSUE #185: Update Donation Link in `About` Page

### PR DESCRIPTION
**Pull request by prestosole**
I am baffled as to why the `pubspec.lock` file got modified.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Download [here](https://github.com/PrestoSole/clima/releases) or get it on `F-dr
 
 ## Donations
 
-<noscript><a href="https://liberapay.com/Lacerte/donate"> <img alt="Donate using Liberapay" src="https://liberapay.com/assets/widgets/donate.svg"></a> </noscript><img src="https://img.shields.io/liberapay/receives/Lacerte.svg?logo=liberapay"> <img src="https://img.shields.io/liberapay/goal/Lacerte.svg?logo=liberapay">
+<noscript><a href="https://liberapay.com/lacerte/donate"> <img alt="Donate using Liberapay" src="https://liberapay.com/assets/widgets/donate.svg"></a> </noscript><img src="https://img.shields.io/liberapay/receives/lacerte.svg?logo=liberapay"> <img src="https://img.shields.io/liberapay/goal/lacerte.svg?logo=liberapay">
```
Comment (triallax) [inline README.md:22]: What are these changes for?

Comment (prestosole) [reply README.md:22]: The rest of the links are in lowercase, so I just changed those to make them all alike.

Comment (triallax) [reply README.md:22]: To be honest, I don't see why it should be lowercase; the official name is capitalized, even in links. However, that's a nitpick that has no impact on behavior, so you can ignore it.

**Comment by triallax** [comment]
> I am baffled as to why the pubspec.lock file got modified.

Maybe you ran `flutter run` or switched branches without `git checkout --`ing `pubspec.lock`.

**Comment by triallax** [comment]
In any case, please remove the lockfile changes.

**Comment by prestosole** [comment]
How?

**Review by triallax — APPROVED** [review]

**Pull request merged**